src/app/controllers/login/login.component.ts

```diff
@@ -7,7 +7,6 @@ import {Title} from "@angular/platform-browser";
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
-  styleUrls: ['./login.component.scss']
```

Comment:
Shouldn't this still be here?

Reply:
I guess it doesn't matter if it's blank . . .

Reply:
Did you remove that code when you merged? Cause it was supposed to be there. . .